File: testdata/csharp/edge_cases/broken.cs
// Intentionally malformed: tree-sitter parses this with ERROR nodes.
// The parser must skip error nodes gracefully without panic.

namespace Bad
{
    public class Foo
    {
        public void Bar(
        {
        }

        public void Good() { }
    }

    public class AlsoGood
    {
        public void Run() { }
    }
}

File: testdata/java/edge_cases/Broken.java
// Intentionally malformed: tree-sitter parses this with ERROR nodes.
// The parser must skip error nodes gracefully without panic. The
// recovered symbol count is what tree-sitter-java 0.23.5 actually
// produces — run and record, not zero.

package edge_cases;

public class Broken {
    public void bar(
    {
    }

    public void good() { }
}

class AlsoGood {
    public void run() { }
}
